commands/release.go

```diff
@@ -104,6 +104,9 @@ With '--exclude-prereleases', exclude non-stable releases from the listing.
 	-i, --include <PATTERN>
 		Filter the files in the release to those that match the glob <PATTERN>.
 
+	-T, --tagname <NEW-TAG-NAME>
```

Comment:
Since I don't imagine this flag being used too often, we probably don't need the shorthand `-T`. Also, I'm thinking that the flag might benefit from a more descriptive name, since it might not be clear what `--tagname FOO` will do when editing a release.

What do you think about:
- `--newtag <TAG-NAME>`
- `--edit-tag <NEW-TAG-NAME>`

Reply:
For sure, `-T` does feel unnecessary. I was just trying to be most consistent- I'll remove it.

I like `--newtag`. I think that makes much more sense!